lib/aur-fetch

```diff
@@ -8,88 +8,86 @@ readonly startdir=$PWD
 readonly PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
 
 # default options
-mode=git recurse=0
+log=directory mode=git recurse=0
 
 git_stat_patch() {
     git --no-pager log -O "$1" --patch --stat '..@{upstream}'
 }
 
-merge_upstream() {
-    # set destination of patch file
-    local pkg=$1 log_dir=$2 output
-
-    if [[ -d $log_dir ]]; then
-        output=$log_dir/$pkg.patch
-    else
-        output=/dev/stdout
-    fi
-
-    # default to showing PKGBUILD diffs first (#399)
+merge() {
```

Comment:
Nitpick: Slightly bothers me that we're calling the function `merge` when we're doing a `reset.` 🙃 

This is about getting in sync with upstream so maybe something like sync or upstream_sync?